roles/openshift_node/tasks/systemd_units.yml

```diff
@@ -7,14 +7,29 @@
     docker pull {{ openshift.node.node_image }}:{{ openshift_image_tag }}
   register: pull_result
   changed_when: "'Downloaded newer image' in pull_result.stdout"
-  when: openshift.common.is_containerized | bool
+  when: openshift.common.is_containerized | bool and not openshift.common.is_node_system_container | bool
```

Comment:
@ashcrow no, it is fine, let's fix it now.  I have some WIP for moving the Docker container related stuff into blocks, I am going to clean it up, test and post it later